crates/gpio/src/backend.rs

```diff
@@ -48,18 +69,22 @@ struct GpioArgs {
     socket_count: usize,
 
     /// List of GPIO devices, one for each guest, in the format
-    /// <N1>[:<N2>]. The first entry is for Guest that connects to
-    /// socket 0, next one for socket 1, and so on. Each device number
-    /// here will be used to access the corresponding /dev/gpiochipX.
-    /// or simulate a GPIO device with N pins. Example, "-c 4 -l
-    /// 3:s4:6:s1"
+    /// [s]<N1>[:[s]<N2>].
+    #[cfg(feature = "dummy")]
+    #[clap(short = 'l', long)]
+    device_list: String,
+
+    /// List of GPIO devices, one for each guest, in the format
+    /// <N1>[:<N2>].
+    #[cfg(not(feature = "dummy"))]
     #[clap(short = 'l', long)]
     device_list: String,
```

Comment:
I don't see a difference in the above two device_list fields here. what's changing based on the feature here ?

Reply:
I think this is just to ensure we get the right --help text depending on the build config.